text-utilities.css

```diff
@@ -0,0 +1,18 @@
+heading-size-large {
```

Comment:
I thing this should be a class. So a dot will come.
.heading-size-large {
.......
}

Reply:
Done